composer.json

```diff
@@ -10,10 +10,9 @@
         }
     ],
     "require": {
```

Comment:
Please also require and enable:
https://www.drupal.org/project/navigation_extra_tools

It has no config.